packages/core/core/src/AssetGraphBuilder.js

```diff
@@ -298,96 +233,81 @@ export default class AssetGraphBuilder extends EventEmitter {
     await Promise.all(promises);
   }
 
-  queueRequest(request: AssetGraphBuildRequest, runOpts: RunRequestOpts) {
-    return this.queue.add(async () => {
-      if (this.rejected.size > 0) {
-        return;
-      }
-
-      try {
-        await this.runRequest(request, runOpts);
-      } catch (e) {
-        this.rejected.set(request.id, e);
-      }
-    });
+  shouldSkipRequest(node: AssetGraphNode) {
+    const typesWithRequests = [
```

Comment:
Hoist this out of the class to a constant and make it a set for faster inclusion checking?